src/main/java/io/stargate/sgv2/jsonapi/service/resolver/update/TableUpdateResolver.java

```diff
@@ -1,10 +1,14 @@
 package io.stargate.sgv2.jsonapi.service.resolver.update;
 
+import static io.stargate.sgv2.jsonapi.exception.ErrorFormatters.*;
+
 import com.datastax.oss.driver.api.core.CqlIdentifier;
 import com.fasterxml.jackson.databind.node.ObjectNode;
 import io.stargate.sgv2.jsonapi.api.model.command.Command;
 import io.stargate.sgv2.jsonapi.api.model.command.CommandContext;
 import io.stargate.sgv2.jsonapi.api.model.command.Updatable;
+import io.stargate.sgv2.jsonapi.api.model.command.clause.filter.JsonLiteral;
+import io.stargate.sgv2.jsonapi.api.model.command.clause.filter.JsonType;
```

Comment:
comments for things below...

remove supportedOperators() it is used, update the comments for the supportedOperators var as appropriate 

if update clause is null or empty, a APIExecption ZERO_UPDATE_OPERATIONS and include the list of supported update ops 

if (resolverFunction == null)  - we want to throw a APIExecption UNSUPPORTED_UPDATE_OPTION_FOR_TABLE include the list of supported update ops 

remove UnvalidatedClauseException 

to make the CqlIdentifier use the from user input function 

remove the TODO comment , it sOK

Reply:
fixed